android/src/main/java/com/robinpowered/AudioJack/AudioJackModule.java

```diff
@@ -74,8 +74,12 @@ private boolean isHeadsetPluggedIn() {
             AudioDeviceInfo[] devices = audioManager.getDevices(AudioManager.GET_DEVICES_OUTPUTS);
             for (int i = 0; i < devices.length; i++) {
                 AudioDeviceInfo device = devices[i];
-                if (device.getType() == AudioDeviceInfo.TYPE_WIRED_HEADPHONES) {
-                    return true;
+                switch(device.getType()) {
+                    case AudioDeviceInfo.TYPE_WIRED_HEADSET:
+                    case AudioDeviceInfo.TYPE_WIRED_HEADPHONES:
+                    case AudioDeviceInfo.TYPE_BLUETOOTH_SCO:
```

Comment:
Adding `TYPE_WIRED_HEADSET` LGTM but this library is meant for determining whether a wired audio device is plugged in, as described in the package name. Also, adding this means having an inconsistent API between iOS and Android since the iOS version only looks for `AVAudioSessionPortHeadphones`.

Will defer to active maintainers @christophermark

Reply:
For now i can remove the bluetooth types in order for a merge? :)

Reply:
@yinghang is right - adding `WIRED_HEADSET` makes sense, as it keeps with the API.

If you want to detect Bluetooth types, feel free to add a separate getter method to return those (in a separate PR).

If you remove bluetooth methods, we can merge 😄

Reply:
I'd maybe make this opt-in,  through a boolean argument. Something like "includeBluetoothDevices" and default to false. That way it's backwards compatible and opt-in, and you don't neglect the name of the module.

Reply:
It's Chris' call but someone will still have to implement the same thing for iOS

Reply:
That's why I suggested this. That makes the transition easier once iOS gets added. I can add that actually once I implement the headphone detection for my app, I wouldn't mind.

Reply:
@christophermark Bluetooth types have been removed!